models/user/user.go

```diff
@@ -584,26 +585,23 @@ var (
 		"explore",
 		"favicon.ico",
 		"ghost",
-		"help",
-		"install",
 		"issues",
-		"less",
 		"login",
 		"manifest.json",
 		"metrics",
 		"milestones",
 		"new",
 		"notifications",
 		"org",
-		"plugins",
```

Comment:
It was imho reserved for future use

Reply:
It's better to use the char `~` for all new paths in future, then we do not need to reserve anything more.